src/Raven.Client/Documents/Indexes/PutIndexResult.cs

```diff
@@ -12,5 +12,7 @@ public class PutIndexResult
         /// Gets or sets the total results for this query
         /// </summary>
         public string Index { get; set; }
+
+        public long RaftIndex { get; set; }
```

Comment:
It is called 'RaftCommandIndex' elsewhere? Why do you need this?

Reply:
changed. 
we need this so we can wait for the index creation (in the cluster) before checking it for staleness.

Reply:
Why you expose this to the Client? Indexes are already created after execution of CreateIndex(Internal) ?